packages/inspector/src/add/add-workflow.ts

```
@@ -150,82 +108,108 @@ export const addWorkflow: AddWiring = (
+        )
+        simple = false
+        getWorkflowInvocations(
+          resolvedFunc,
+          checker,
+          state,
+          workflowName,
+          steps
+        )
+      }
     }
+  } else if (resolvedFunc) {
```

Biome (2.1.2)

[error] This branch can never execute. Its condition is a duplicate or covered by previous conditions in the if-else-if chain.



(lint/suspicious/noDuplicateElseIf)